packages/ui/src/NumberField.tsx

```diff
@@ -28,7 +28,7 @@ export type NumberFieldExtraProps = {
   className?: string
```

Comment:
I found a bug in `NumberField` not related to this PR changes.
When you remove the value from the number field, you get this error: 
![image](https://user-images.githubusercontent.com/17862968/99787746-6ba73480-2b20-11eb-9884-2b3e5aba3cc3.png)

In the component we try to parse the input value with `parseInt` or `parseFloat`. However we do not deal with an empty string `''`. If you try to parse an empty string, you get `NaN`.

Reply:
Also we should specify plus and minus `IconButton` prop `type='button'`, otherwise we're getting this warning:
![image](https://user-images.githubusercontent.com/17862968/99788416-5d0d4d00-2b21-11eb-86cc-bbe1b0803922.png)

Reply:
Fix in https://github.com/hazelcast/frontend-shared/pull/82